lib/test/clojure_lsp/features/file_management_test.clj

```diff
@@ -5,7 +5,9 @@
    [clojure-lsp.shared :as shared]
    [clojure-lsp.test-helper :as h]
    [clojure.test :refer [are deftest is testing]]
-   [medley.core :as medley]))
+   [medley.core :as medley]
+   [clojure-lsp.feature.rename :as f.rename]
+   [clojure-lsp.queries :as q]))
```

Comment:
I wonder how lint didn't catch this on `clean-ns` 🤔 maybe it's not running for tests...

Reply:
Good question... I'll run clean-ns manually, but it'd be nice for the lint CI to catch that.

Reply:
Nm... you're too fast!